lib/index.js

```diff
@@ -133,6 +133,10 @@ var Horseman = function Horseman(options) {
 		}
 	}
 
+    Object.keys(this.options.phantomOptions || {}).forEach(function (key) {
```

Comment:
The indentation here does not match the surrounding indentation, and I am neurotic.